components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/gatewayartifactsynchronizer/DBRetriever.java

```diff
@@ -60,6 +64,11 @@ public void init() throws ArtifactSynchronizerException {
     public String retrieveArtifact(String APIId, String gatewayLabel, String gatewayInstruction)
             throws ArtifactSynchronizerException {
         CloseableHttpResponse httpResponse = null;
+        try {
+            Thread.sleep(gatewayArtifactSynchronizerProperties.getEventWaitingTime());
+        } catch (InterruptedException e) {
+            // Ignore
```

Comment:
Log the exception here.

Reply:
https://github.com/wso2/carbon-apimgt/pull/9016/commits/bb95666b85e9ca2fe650f54b212bcdce5d06008f